reflection.go

```diff
@@ -0,0 +1,37 @@
+package grpcutil
```

Comment:
maybe this should also go in the middleware file since it's using the authless mixin similar to the health server

Reply:
I don't think so. It's _using_ middleware, but it's not implementing a middleware. At least that was my logic.